input/fsh/examples/encounter.fsh

```diff
@@ -4,8 +4,8 @@ Usage: #example
 Description: "An example ManaakiNgaTahi Encounter"
 * status = #in-progress
 * class = http://terminology.hl7.org/CodeSystem/v3-ActCode#VR "virtual"
-* serviceType = http://terminology.hl7.org/CodeSystem/service-type#124 "General Practice"
-* serviceType.text = "General Practice"
+// * serviceType = http://terminology.hl7.org/CodeSystem/service-type#124 "General Practice"
```

Comment:
Any reason for removing `serviceType` from the example?

Reply:
I can't remember why I did this but I was probably experimenting with ManaakiTahiEncounter before profiling a new RF the resource from base Encounter.  I have reverted these lines as per 2 months ago version.